apps/web-app/src/components/PageContainer.tsx

```diff
@@ -2,9 +2,10 @@
 
 import { useLogContext } from "@/context/LogContext"
 import shortenString from "@/utils/shortenString"
-import { Container, HStack, Icon, IconButton, Link, Spinner, Stack, Text } from "@chakra-ui/react"
+import { Container, HStack, Icon, IconButton,Spinner,Link, Stack, Text } from "@chakra-ui/react"
```

Comment:
There is no need to update this line, could you keep it like it was? `import { Container, HStack, Icon, IconButton, Link, Spinner, Stack, Text } from "@chakra-ui/react"`

Reply:
i dont have yarn installed ,what to do ?

Reply:
Hey! You don't need to install yarn, there is a local version. Let me know if you see any errors after running `yarn && yarn prettier:write`. If there are no errors and the process finishes successfully, feel free to push the changes.